models/subnet-group.ts

```diff
@@ -1,6 +1,11 @@
+export interface ChainGroup {
```

Comment:
How do we plan on handling a chain with different numbers of mainnet and testnets? Or will they always be 1 to 1?